updater/fetchers/rhel/rhel_test.go

```diff
@@ -41,24 +44,33 @@ func TestRHELParser(t *testing.T) {
 		expectedFeatureVersions := []database.FeatureVersion{
 			{
 				Feature: database.Feature{
-					Namespace: database.Namespace{Name: "centos:7"},
-					Name:      "xerces-c",
+					Namespace: database.Namespace{
+						Name:          "centos:7",
+						VersionFormat: "rpm",
+					},
+					Name: "xerces-c",
 				},
-				Version: types.NewVersionUnsafe("3.1.1-7.el7_1"),
+				Version: "0:3.1.1-7.el7_1",
```

Comment:
The problem here is that we're comparing the objects with testify. It's doing a straight up string comparison. I think it's best to leave the epoch on because that's how it was presented in the source.